CMakeLists.txt

```diff
@@ -2605,9 +2605,8 @@ find_package(LibUSB)
 # USB HID controller support
 option(HID "USB HID controller support" ON)
 if(HID)
-  find_package(hidapi)
```

Comment:
0.10.1 is the minimum required version, i.e. `0.10.1<` is what we actually want here. If a newer version breaks the build an upper bound could be introduced as a temporary workaround before finishing the upgrade.

Unfortunately, version ranges are only supported starting with CMake 3.19: https://cmake.org/cmake/help/latest/command/find_package.html

Reply:
That is not valid:
```
CMake Error at CMakeLists.txt:2608 (find_package):
  find_package called with invalid argument "0.10.1<"
```

Reply:
Probably `0.10.1...<`
https://cmake.org/cmake/help/latest/command/find_package.html

Reply:
No, that does not work either:
```
CMake Error at CMakeLists.txt:2608 (find_package):
  find_package called with invalid argument "0.10.1...<"
```

I presume anything that would match `if(VERSION_GREATER_EQUAL)` would work, but the CMake documentation is not explicit about this. If you want to specify only one specific version, the documentation says to use EXACT.

Reply:
I have misunderstood the documentation. You can either specify a single (minimum?) version or both.